Modules/System/AL Rest Client/src/ALRestClientImpl.Codeunit.al

```diff
@@ -0,0 +1,153 @@
+codeunit 2351 "AL Rest Client Impl."
+{
+    Access = Internal;
+    InherentEntitlements = X;
```

Comment:
Please add `InherentEntitlements` and `InherentPermissions` properties to all the other objects as well. There is no permission requirement to use the built-in HttpClient, this functionality should not be different.

Reply:
Added to every codeunit.